### PR TITLE
Add desiredAirflowVersion key & updatedDeploymentAirflow Mutation

### PR DESCRIPTION
**Pull request by samblackk**
## Description

Initializes the key for `desiredAirflowVersion` and mutation to update `updateDeploymentAirflow` in order to support managing the Airflow version from the UI and CLI.

## 🎟 Issue(s)

Apart of https://github.com/astronomer/issues/issues/1810

## 🧪 Functional Testing

Can test in the playground.

Query: 
```
query workspaceDeployment (
    $workspaceUuid: Uuid!
    $releaseName: String!
  ) {
    workspaceDeployment (
      workspaceUuid: $workspaceUuid
      releaseName: $releaseName
    ) {
      id
      airflowVersion
      desiredAirflowVersion
    }
  }
```

Mutation:
```
mutation updateDeploymentAirflow (
    $releaseName: String!
    $desiredAirflowVersion: String!
  ) {
    updateDeploymentAirflow (
      releaseName: $releaseName
      desiredAirflowVersion: $desiredAirflowVersion
    ) {
      id
      airflowVersion
      desiredAirflowVersion
    }
  }
```

## 📸 Screenshots

Screen Shot 2020-09-28 at 3 54 16 PM
Screen Shot 2020-09-28 at 3 54 11 PM
Screen Shot 2020-09-28 at 3 54 21 PM

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `yarn lint && yarn test` before taking out of draft
- [x] Added/updated applicable tests
- [ ] Tested against [Astro-UI](https://github.com/astronomer/astro-ui/) and [Astro-CLI](https://github.com/astronomer/astro-cli/) clients.
- [x] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: prisma/migrations/20200928192950/steps.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "steps": [
+    {
+      "tag": "CreateField",
+      "model": "Deployment",
+      "field": "desiredAirflowVersion",
+      "type": "String",
+      "arity": "Optional"
+    }
+  ]
+}
```
Comment (andriisoldatenko) [inline prisma/migrations/20200928192950/steps.json:12]:
```suggestion
}

```

**Comment by andriisoldatenko** [comment]
@samblackk LGTM!
it's first prisma2 PR ❤️ !

**Review by andriisoldatenko — APPROVED** [review]

File: src/resolvers/mutation/update-deployment-airflow/index.unit.test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    ) {
+        id
+        airflowVersion
+        desiredAirflowVersion
+      }
+    }
+`;
+
+describe("updateDeploymentAirflow", () => {
+  test("typical request is successful", async () => {
```
Comment (adam2k) [inline src/resolvers/mutation/update-deployment-airflow/index.unit.test.js:24]: Is there also a failure case here?  They try to update to an out of date version of Airflow or invalid version of Airflow?

**Review by adam2k — APPROVED** [review]
This looks good to me 🎉.  I added a comment about some extra failure test cases, but other than that I think this will work!

File: src/resolvers/mutation/update-deployment-airflow/index.unit.test.js (hunk 2 above)
Comment (samblackk) [reply src/resolvers/mutation/update-deployment-airflow/index.unit.test.js:24]: I'm adding the validation now - will update w/ test cases shortly.

**Comment by samblackk** [comment]
@adam2k Validation & test updated :)

**Pull request merged**